docs/bookmarks.md

```diff
@@ -11,8 +11,8 @@ pass a bookmark's name to commands that want a revision as argument. For example
 `jj bookmark list` to list bookmarks and `jj bookmark` to create, move, or delete
 bookmarks. There is currently no concept of an active/current/checked-out bookmark.
 
-Currently Jujutsu maps its Bookmarks to Git Branches and stores them as that 
-in the Git backend. This means that all Bookmarks will be reflected as 
+In a Jujutsu repo that's colocated with Git, Jujutsu's bookmarks map to Git 
+branches. This means that all Bookmarks will be reflected as 
```

Comment:
I thought that it was true as soon as git is used as a backend, even in non-colocated repositories. Am I wrong?

Reply:
its based on this comment https://github.com/martinvonz/jj/pull/4341#discussion_r1747911931.

Reply:
> I thought that it was true as soon as git is used as a backend, even in non-colocated repositories. Am I wrong?

You're right. Well, it was more correct before than after at least, but the old form also sounds a bit incorrect to me. I think "stores them as that in the Git backend" makes it sound like we actually store bookmarks as branches in the Git backend. We store bookmarks in the operation store (which has no Git backend). We have `jj git export/import` to export/import from/to our operation store to Git. The export/import requires a Git backend, but it is not the source of truth for storage of bookmarks.